src/Domain/Regulation/Repository/RegulationOrderRecordRepositoryInterface.php

```diff
@@ -13,8 +13,9 @@ public function add(RegulationOrderRecord $regulationOrderRecord): RegulationOrd
 
     public function findOneByUuid(string $uuid): ?RegulationOrderRecord;
 
-    public function findRegulationsByOrganizations(
-        array $organizations,
+    public function findAllRegulations(
```

Comment:
La fonction est-elle bien nommée ?
J'ai pas fait de condition sur l'`array $organizations` (cas où il est vide), la requête DQL va-t-elle s'en sortir ?

Reply:
C'est vrai que c'est toujours une ambigüité entre null et un tableau vide

En l'occurrence tu traites les deux cas pareil grâce à `if $organizationUuids` donc y'a pas trop d'enjeu je dirais